### DISCUSSION TITLE
Synology Docker issues

### DISCUSSION THREAD

**Discussion by BarterClub** [FoxxMD/context-mod#70] [Q&A]
I'm having issues staring at the bot on my Synology docker container. When trying to look at the web interface it won't work. Tried with my IP of the server and localhost.

**Comment by FoxxMD** [comment]
Hi there! I'm not too familiar with how synology handles docker.

* Do you have any logs output you can share?
* Are you correctly binding the container's web port? The container listens on port `8085`

**Reply by BarterClub** [reply]
The first is the docker variables

https://jmp.sh/LJMZ72z

The next is the logs.

https://jmp.sh/BYUSNFa

**Reply by FoxxMD** [reply]
The container is running correctly. It sounds like you have not configured the bridge network for the container. I found these links that may help you: 

https://www.wundertech.net/how-to-use-docker-on-a-synology-nas/

^^ This article explains what you need. You need to bind a volume to a folder on your synology host in order to persist the configuration file. You also need to make sure the **Bridge Network** is setup and that you have port forwarding (??) or mapping between local port and container setup for this container.

https://forums.docker.com/t/help-forwarding-ports-within-a-container/98572
https://stackoverflow.com/questions/52571351/how-do-i-make-synology-diskstation-port-visible-to-computer-for-docker-app
